modules/heat_conduction/test/tests/gap_heat_transfer_htonly/gap_heat_transfer_htonly_test.i

```diff
@@ -153,9 +153,10 @@
     thermal_conductivity = 100000000.0
   [../]
   [./density]
-    type = Density
+    type = GenericConstantMaterial
```

Comment:
What's the deal with this substitution?

Reply:
Oh, I see, this allows you to move it out of combined and into heat_condution!

Reply:
Yup! exactly. There are a lot of these types of tests in combined, that I have been slowly trying to migrate back to their respective modules whenever density is a constant. There are not even more improperly placed tests now that solid mechanics is gone.